pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint

```diff
@@ -1,32 +1,34 @@
-In the connection from test-vpc1-ky/vsi2-ky[10.240.20.4] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+In the connection from "test-vpc1-ky/db-endpoint-gateway-ky[10.240.30.7]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP response is blocked
+In the connection from "test-vpc1-ky/vsi1-ky[10.240.10.4]" to "Public Internet 161.26.0.0/16" TCP response is blocked
+In the connection from "test-vpc1-ky/vsi2-ky[10.240.20.4]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP response is blocked
 ... (3 more)
 
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "NACL for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-network ACL demilune-humorless-captain-lurex of VPC test-vpc1-ky has no resources attached to it
+network acl "demilune-humorless-captain-lurex" of VPC "test-vpc1-ky" has no resources attached to it
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
-"SG for which there are no resources attached to" issues:
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-security group barbecue-frayed-varied-average of VPC test-vpc1-ky has no resources attached to it
+"NACLs implying different connectivity for endpoints inside a subnet" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In VPC "test-vpc1-ky", network ACL "acl3-ky" rule [1] splits subnet "subnet3-ky" (10.240.30.0/24).
+	Rule's details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC "test-vpc1-ky", network ACL "acl3-ky" rule [3] splits subnet "subnet3-ky" (10.240.30.0/24).
+	Rule's details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
-"rules of network ACLs implying different connectivity for different endpoints within a subnet" issues:
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 1 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
-In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 3 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
+"SG for which there are no resources attached to" issues:
```

Comment:
I think this phrasing is mine, but I'm reading it again it is clunky. How about

"Security Groups not applied to any resources"

Reply:
also "NACL not applied to any resources" ?
@zivnevo agreed?

Reply:
@zivnevo ? (`unattached` was agreed today with him)

Reply:
fixed for both